### PR TITLE
css styles in class, task-done-method working, deadline data is showi…

### PR DESCRIPTION
**Pull request by kasiaekiert**
css styles in class, task-done-method working, deadline data is showi…

### CODE REVIEW

File: app/controllers/tasks_controller.rb [hunk 1]
```diff
@@ -64,15 +57,19 @@ def destroy
     end
   end
 
+  def toggle_done
+    binding.pry
```
Comment (koszany) [inline app/controllers/tasks_controller.rb]: I guess this is pushed accidently

File: app/views/tasks/index.html.erb [hunk 2]
```diff
@@ -23,10 +23,10 @@
     <tbody>
       <% @tasks.each do |task| %>
         <tr scope="row">
-          <td style="width: 15%">
+          <td class= "width15">
             <%= task.name %>
           </td>
-          <td style="width: 25%">
+          <td class= "width15" >
```
Comment (koszany) [inline app/views/tasks/index.html.erb]: I'd keep inline css or give more meaningful class name

File: app/controllers/tasks_controller.rb [hunk 3]
```diff
@@ -64,15 +57,19 @@ def destroy
     end
   end
 
+  def toggle_done
```
Comment (koszany) [inline app/controllers/tasks_controller.rb]: it can be actually used as well to undone the task

File: app/views/tasks/index.html.erb (hunk 2 above)
Comment (kasiaekiert) [reply app/views/tasks/index.html.erb]: corrected

**Review by koszany — APPROVED** [review]

**Pull request merged**